### PR TITLE
HUD health bars will now appear in the top left for exceptionally large mobs

### PR DESCRIPTION
**Pull request by ChangelingRain**
Like bubblegum and legion and suchlike.
Megacarp now have their own DMI because they'd have a really weird healthbar spot otherwise.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/carp.dm [hunk 1]
```diff
@@ -51,7 +51,7 @@
 	del_on_death = 1
 
 /mob/living/simple_animal/hostile/carp/megacarp
-	icon = 'icons/mob/alienqueen.dmi'
+	icon = 'icons/mob/64x32.dmi'
```
Comment (Iamgoofball) [inline code/modules/mob/living/simple_animal/hostile/carp.dm]: Stop naming DMIs this, it'll break with 64x64

Comment (Jordie0608) [inline code/modules/mob/living/simple_animal/hostile/carp.dm]: More importantly it makes it hard to know whats inside at a glance.

File: code/modules/mob/living/simple_animal/hostile/carp.dm [hunk 2]
```diff
@@ -51,7 +51,7 @@
 	del_on_death = 1
 
 /mob/living/simple_animal/hostile/carp/megacarp
-	icon = 'icons/mob/64x32.dmi'
+	icon = 'icons/mob/64x32mobs.dmi'
```
Comment (Iamgoofball) [inline code/modules/mob/living/simple_animal/hostile/carp.dm]: No, the problem is set dimensions in the file name

**Comment by MrPerson** [comment]
Wouldn't it make more sense to just use bounds_width x bounds_height as the mob size?

File: code/modules/mob/living/simple_animal/hostile/carp.dm (hunk 2 above)
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/hostile/carp.dm]: Basically unfixable find me a descriptor for 'wide but not tall' mobs and you will be considered.

**Comment by ChangelingRain** [comment]
No, because all mobs have a bounds_width/height of 32; it's the collision size, not the icon size.

File: code/modules/mob/living/simple_animal/hostile/carp.dm (hunk 2 above)
Comment (optimumtact) [inline code/modules/mob/living/simple_animal/hostile/carp.dm]: broadmobs

You're welcome

**Pull request merged**